fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,44 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on label `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    the event corresponding to `y=1`.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
```

Comment:
The class of `prob_event` is `pandas.Series` not `pandas.DataFrame`. I also think that we can just drop the class designation here.

Reply:
I will remove it here. But the `prob_event` is a `DataFrame`  when I checked. I think it would be better to just remove the class designation as you suggested. Will do it 👍